node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -23,6 +35,7 @@ pub const INFINITY_PUBLIC_KEY: [u8; PUBLIC_KEY_BYTES_LEN] = [
 ];
 
 /// Type safety wrapper around a `blst` SecretKey
+#[derive(ZeroizeOnDrop)]
```

Comment:
It seems that blst [zeroizes](https://github.com/supranational/blst/blob/master/bindings/rust/src/lib.rs#L550) its `SecretKey`, but let's keep it here too to know that we zeroize the key